apis/swagger.yml

```diff
@@ -3597,6 +3597,49 @@ definitions:
           items:
             type: "string"
 
+  EventsMessage:
```

Comment:
Please also add the route definition. like:
```
 /containers/{id}/stop:
```

Reply:
i have split this whole feature into 4 parts, the api will be added at API pr

Reply:
@allencloud more information can refer: https://github.com/alibaba/pouch/issues/2052